postgres/pgdump.go

```diff
@@ -620,6 +659,19 @@ func toSchemaKeys(conv *internal.Conv, table string, s []string) (l []schema.Key
 	return l
 }
 
+// toIndexKeys converts a string list of PostgreSQL primary keys to
+// schema Index keys.
+func toIndexKeys(conv *internal.Conv, table string, s []nodes.Node) (l []schema.Key) {
```

Comment:
Let's get rid of named return parameter as discussed before.

Reply:
updated return type